ci/workflows/pull-request.yml

```diff
@@ -333,6 +333,25 @@ steps:
     timeout_in_minutes: 10
     retry: *auto-retry
 
+  - label: "build-image-on-pr"
+    key: "build-image-on-pr"
+    if: build.pull_request.labels includes "build-image-on-pr" || build.env("CI_STEPS") =~ /(^|,)build-image-on-pr?(,|$$)/
```

Comment:
```suggestion
    if: build.pull_request.labels includes "build-image-on-pr" || build.env("CI_STEPS") =~ /(^|,)build-image-on-pr(,|$$)/
```

Reply:
You might have tried to write the regex as "...sink_tests?(,|$$)/"
However, the "?" here means the "s" in "sink_test[s]" is optional.
So please remove this "?"